src/server/pfs/fuse/fuse_test.go

```diff
@@ -40,18 +44,48 @@ func TestBasic(t *testing.T) {
 	})
 }
 
+func TestChunkSize(t *testing.T) {
+	c := server.GetPachClient(t)
+	require.NoError(t, c.CreateRepo("repo"))
+	_, err := c.PutFile("repo", "master", "file", strings.NewReader(strings.Repeat("p", int(pfs.ChunkSize))))
+	require.NoError(t, err)
+	mount(t, c, nil, func(mountPoint string) {
+		data, err := ioutil.ReadFile(filepath.Join(mountPoint, "repo", "file"))
+		require.NoError(t, err)
+		require.Equal(t, int(pfs.ChunkSize), len(data))
+	})
+}
+
 func TestLargeFile(t *testing.T) {
 	c := server.GetPachClient(t)
 	require.NoError(t, c.CreateRepo("repo"))
-	_, err := c.PutFile("repo", "master", "file", strings.NewReader(strings.Repeat("p", GB)))
+	src := workload.RandString(rand.New(rand.NewSource(123)), GB+17)
```

Comment:
Just curious about the GB threshold ... I'm guessing you wanted to test something just past 1GB and the `17` is an arbitrary small number to get past 1GB

Reply:
Yup that's the intention here. FUSE sends requests for 128K chunks so I wanted something not divisible by that so we'd get some differently sized requests at the end.

Reply:
:+1: